### ISSUE TITLE
[SUGGESTION] Add a heavily configurated Agricraft

### ISSUE THREAD

**Issue by joendter** [Gliese-832-c/ad-astra-per-nucleon#22]
**Is your suggestion related to a problem? Please describe.**
In this modpack radiation and space exploration are important. So are crops to sustain food-production. 

**Describe the solution you'd like**
[Agricraft](https://www.curseforge.com/minecraft/mc-mods/agricraft) adds "mutations" and crossing of plants. Maybe you could modify it in a way that radiation affects the crop mutation and crossing, or even the likelihood for the crossover to end in weeds. I don't know how much effort this would be (probably a lot). Also maybe consider the ore plants for midgame renewable resources or disable them. Because it's compatible with Pam's Harvestcraft and somewhat with Emerging Technology i think it would fit in just fine.

**Comment by Gliese-832-c** [comment]
Firstly, as you said, it would be a lot of effort. Secondly, I don't even know if it's possible. Assuming that NBT data radiation block mutations are possible, it would be hundreds of permutations of crops. I'll leave it open for now until I look at it in detail.

**Comment by joendter** [comment]
Yeah just try it if you have nothing better to do (the modpack is done)...
